controllers/factory/vmcluster.go

```diff
@@ -399,6 +399,8 @@ func makePodSpecForVMSelect(cr *v1beta1.VMCluster, c *config.BaseOperatorConf) (
 		args = append(args, fmt.Sprintf("-loggerFormat=%s", cr.Spec.VMSelect.LogFormat))
 	}
 	if cr.Spec.ReplicationFactor != nil && *cr.Spec.ReplicationFactor > 1 {
+		args = append(args, fmt.Sprintf("-replicationFactor=%d", *cr.Spec.ReplicationFactor))
```

Comment:
I think we should also allow user to override `-replicationFactor` using cr.Spec.VMSelect.ExtraArgs, since vmselect can be set up to query other vmstorages with different settings.

Reply:
Done

Reply:
> According to [this comment](https://github.com/VictoriaMetrics/operator/issues/778#issuecomment-1744695815), should we add `-replicationFactor` only when vmselects>=[v1.91.3](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.91.3) by default. For lower version, users can add it by hand.

It's a pretty old version, I can't imagine a case where users roll back so far from the current VM version in an operator - a lot of things can break, they usually upgrade to a higher version.